src/Microsoft.ML.Core/Utilities/MathUtils.cs

```diff
@@ -257,11 +257,7 @@ public static Float SoftMax(Float[] inputs, int count)
 
             if (count == 1)
                 return max;
-
-            //else if (leng == 2) {
-            //  return SoftMax(inputs[0], inputs[1]);
-            //}
-
+            
```

Comment:
Blank line with space on it introduced.

Reply:
Is the blank line okay as long as the whitespace is removed?That matches the if statement above.

Also, for my own reference, how can you tell there is whitespace? Is it through Visual Studio? Or is there also a way to know through github?